src/app/components/add-volunteers/add-volunteers.component.ts

```diff
@@ -52,80 +57,63 @@ export class AddVolunteersComponent implements OnInit {
       }),
       numberOfVolunteers: ['', Validators.min(1)],
       availableContent: [''],
-      days: this.fb.group({
-        1: [false],
-        2: [false],
-        3: [false],
-        4: [false],
-        5: [false],
-        6: [false],
-        7: [false],
-      }),
-      regions: this.fb.array(this.regions.map(region => this.fb.control(false))),
+      multiOccurrence: [''],
+      regions: [null],
     });
   }
 
   sendData() {
     const formValues = this.myForm.value;
     const volunteeringOffer: VolunteeringOffer = this.formValues2volunteeringOfferModel(formValues);
 
-    console.log(volunteeringOffer);
-
     this.vltrOffer.create(volunteeringOffer).then(volunteeringOfferDocument => {
-      console.log(volunteeringOfferDocument);
       if (volunteeringOfferDocument) {
         this.router.navigateByUrl('/dashboard');
       }
     });
   }
 
-  regionCheckboxsToRegionArray(regionCheckboxArray: [boolean]) {
-    const availableRegions: [string] = [] as any;
-    regionCheckboxArray.forEach((regionChecked, i) => {
-      if (regionChecked) {
-        availableRegions.push(this.regions[i]);
-      }
-    });
-
-    return availableRegions;
-  }
-
   formValues2volunteeringOfferModel(formValues): VolunteeringOffer {
+    console.log('forms values: ', formValues);
     const volunteeringOffer: VolunteeringOffer = {} as any;
     volunteeringOffer.title = formValues.title;
     volunteeringOffer.contact = formValues.poc;
     volunteeringOffer.numberOfVolunteers = formValues.numberOfVolunteers;
-    volunteeringOffer.availableContent = formValues.availableContent;
-    volunteeringOffer.availableWeekdays = this.daysCheckboxsToArray(formValues.days);
-    volunteeringOffer.regions = this.regionCheckboxsToRegionArray(formValues.regions);
+    volunteeringOffer.content = formValues.availableContent;
+    if ( formValues.multiOccurrence === '' ) {
```

Comment:
Don't wrap the condition with whitespaces

Reply:
בוטל התנאי בעקבות שינוי אחר.